rust/utils/run-locally/src/main.rs

```diff
@@ -256,6 +243,25 @@ fn main() -> ExitCode {
         .status()
         .expect("Failed to run prettier from top level dir");
 
+    // The rust binaries should be built after contracts are deployed
```

Comment:
The build process does not use the configs, right? It is only once they are built it is in use so I don't think this change is necessary.